### ISSUE TITLE
PLY export refuses a finished MRNF model because Gaussians have opacity logit = +inf (opacity_decay = 0); count grows during training

### ISSUE THREAD

**Issue by JLMVFX** [MrNeRF/LichtFeld-Studio#2079]
**v0.5.3-539 (fe7d9029), Windows 11, RTX PRO 6000 Blackwell, driver 596.36.**

Store interior, 1,736 images @ 8192x5464, MRNF, SH 3, 36M cap (reached at iter 102k), Exposure Correction on, **Opacity Decay 0** (Grow Fraction 0.12, everything else default). The run completed all 307,500 iterations (one resume from the 175k checkpoint after a driver TDR, separate report).

`File > Export > PLY` on that loaded model fails immediately:

```
[error] async_task_manager.cpp:1808 Export failed: PointCloud.opacity contains a non-finite value (flat_index=11467, value=inf, shape=[36000000, 1])
```

Counted on the live model through the MCP Python console (`lf.get_scene().training_model().opacity_raw.isinf().sum_scalar()`): **3 rows at iter 175,000, 366 rows at iter 307,500**, all +inf, all in `opacity_raw`; `scaling_raw`, `means_raw`, `rotation_raw`, `sh0_raw` contain no non-finite values. Finite opacity max is 16.85. Training was unaffected (loss normal, no NaNs), so the model is fine, just not exportable.

Workaround that works: `gaussians_write(field="opacity_raw", indices=<the 366 rows>, values=[16.0]*366)` then export; a re-export after reopening the `.licht` and repeating the write is payload-identical (sha256 checked). The checkpoint itself keeps the infs, so every reopen needs the write again.

Two runs on the same dataset with the default Opacity Decay 0.004 exported fine (raw opacity max 7.05). The only difference in this run is opacity_decay = 0, which suggests a Gaussian's opacity reached exactly 1.0 in float32 somewhere and a subsequent inverse-sigmoid (relocation / split opacity adjustment?) produced +inf.

**Requests**
1. Exporter: treat a non-finite opacity logit as fully opaque (clamp to the max finite logit, or to opacity 1 - eps) instead of refusing the whole export. A +inf logit is a valid "opacity 1.0" splat.
2. MRNF: clamp opacity to `1 - eps` before any inverse-sigmoid so the logit cannot overflow when opacity_decay = 0.

I still have the `.licht` (119 GB, checkpoints every 25k from 25,000 to 307,500), which reproduces this on every reopen, plus the index list of the 366 rows. Happy to test a fix or send either.

**Comment by MrNeRF** [comment]
fixed with #2082, merged now. zero opacity decay keeps finite logits unchanged, and the saturation case no longer creates infinity. export also handles old infinite opacity values without changing the loaded model.

the export fix adds no VRAM, repair is on a private CPU copy. checked CPU/CUDA binary and ASCII exports, zero new device allocations. also checked all 7 latest mipnerf360 reconstructions with infinities injected for the test.
